ksqldb-functional-tests/src/test/java/io/confluent/ksql/test/planned/PlannedTestUtils.java

```diff
@@ -47,7 +49,8 @@ public static boolean isPlannedTestCase(final TestCase testCase) {
 
   public static boolean isNotExcluded(final TestCase testCase) {
     // Place temporary logic here to exclude test cases based on feature flags, etc.
-    return true;
+    final Map<String, Object> props = testCase.properties();
+    return !(boolean) props.getOrDefault(KsqlConfig.KSQL_FOREIGN_KEY_JOINS_ENABLED, false);
```

Comment:
For my own education: What does this do and why do we need it?

Reply:
I see -- that was my follow up question: if we disable the feature by default, how can we test it, ie, how can we enable it for QTT tests?